test/sources/ncdatasets.jl

```diff
@@ -479,6 +488,7 @@ end
         @test metadata(saved)["advection"] == "Lin & Rood"
         @test metadata(saved) == metadata(st) == metadata(ncstack)
         @test all(first(DimensionalData.layers(saved)) .== first(DimensionalData.layers(st)))
+        @test (@allocations write(filename, st)) < 1e6 # writing a rasterseries/stack has no force keyword
```

Comment:
does `force` not work here?

Reply:
No, that's a bug.

Reply:
Would be nice to have a `@test_broken` in the tests for with is the kw PR doesn't fix it. I have a bunch of tests writing stacks a second time with `force=true` and they work.